service/src/main/resources/migrations/v14__backfila.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE registered_backfills
+    ADD COLUMN delete_by timestamp(3) NULL;
```

Comment:
Timestamp has the 2038 problem. Since we are potentially actually storing stuff far in the future, I recommend using DATETIME here

Reply:
Done